runtests.py

```diff
@@ -202,6 +202,14 @@ def main(argv):
             shutil.rmtree(dst_dir)
         extra_argv += ['--cov-report=html:' + dst_dir]
 
+    if args.refguide_check:
+        cmd = [os.path.join(ROOT_DIR, 'tools', 'refguide_check.py'),
+               '--doctests']
+        if args.submodule:
+            cmd += [args.submodule]
+        os.execv(sys.executable, [sys.executable] + cmd)
```

Comment:
Probably better to use subprocess

Reply:
This changes the behavior slightly in my hands when using `subprocess.call([sys.executable] + cmd)`. In particular, the output text includes a few lines about building that I don't see in the original output, but otherwise seems fine.